### ISSUE TITLE
Consider updating the license

### ISSUE THREAD

**Issue by akhmerov** [fralau/mkdocs_macros_plugin#100]
The licensing issues below are largely irrelevant, however I would like to bring those up for consideration.

Firstly, the repository license has an outdated copyright year (2018) and lists @fralau as the only copyright holder. I don't think it's technically possible because copyright assignment has to happen explicitly, and the 14 contributors of the PRs did not assign the copyright.

To fix this I recommend to keep copyright with the contributors and list "mkdocs_macros plugin contributors" as the copyright holder. Of course @fralau deserves all the credit, however this is easier to address by e.g. creating `AUTHORS.md` that would state that the library was developed mainly by @fralau (it doesn't actually need to list anyone else explicitly, unless the contributions are important).

Furthermore, the documentation has a CC-BY-SA license. This may create confusion (it was my "huh" moment when I saw this in the footer of the docs), while, to the best of my knowledge, it does not serve a specific goal. I recommend to change the docs license to MIT as well, in line with the main repo license.

**Comment by fralau** [comment]
I got you said about licenses, and you may have a point. In the real world, however, repositories are generally managed by the person(s) who wrote them. Most issues are thus written in a language that assumes this. 

Why is your post written as if I was a third party?

**Comment by akhmerov** [comment]
> I have, however, a question about form: in the real world, repositories are generally managed by the person(s) who wrote them.

Absolutely: copyright does not imply authority in decision making. As I wrote, this is a technical point and it's about the lack of explicit copyright transfer.

> Then why is your post written with my name repeatedly mentioned as a third party?

Very sorry for giving a wrong/odd impression with this formulation. I used your name when I was discussing how it appears or would appear in the repo and it felt ambiguous to use "you" instead. I realize that you are the likely person to read it.
